src/django/iow/settings.py

```diff
@@ -216,9 +213,8 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 # TODO enable static file hosting
 
-# STATIC_URL = '/static/'
-# STATIC_ROOT = os.path.join(BASE_DIR, "static")
-# STATICFILES_DIRS = ((os.path.join(STATIC_ROOT, "static")),)
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, "static")
```

Comment:
When visiting the site, you should see a demo React app and not the blank screen (you can see what the demo app looks like by visiting the webpack dev server at http://localhost:4545/). The blank screen indicated that some JS files weren't being served properly. If you include the `STATICFILES_DIR` line that was commented out on line 221, it allows the JS files to be loaded properly and for the React app to load from http://localhost:8181/

Reply:
Thanks for catching this. I couldn't figure out if this was necessary or not, and after checking a few different projects it looked like some did and some did not have it, so I figured I'd try without it. Lesson learned.